src/Appwrite/Platform/Modules/Functions/Services/Http.php

```diff
@@ -65,7 +64,6 @@ public function __construct()
         $this->addAction(DownloadDeployment::getName(), new DownloadDeployment());
```

Comment:
Do we still call it `DownloadDeployment` or make it more generic?

Reply:
Same name makes sense to me, we are still downloading deployment's files. Either source code, or build output.